sdk/cognitivelanguage/ai-language-text/samples-dev/autoLanguageDetection.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * In this sample, we use the complementary automatic language detection
+ * with an abstractive summarization task to receive the detected
+ * language of the input document.
+ *
+ * The abstractive summarization feature is part of a gated preview. Access can
+ * be request in {@link https://aka.ms/applyforgatedsummarizationfeatures}.
+ *
+ * @summary return detected language with the task
+ * @azsdk-weight 50
+ */
+
+import {
+  TextAnalysisClient,
+  AzureKeyCredential,
+  AnalyzeBatchAction,
+} from "@azure/ai-language-text";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// You will need to set these environment variables or edit the following values
+const endpoint = process.env["ENDPOINT"] || "<cognitive language service endpoint>";
+const apiKey = process.env["LANGUAGE_API_KEY"] || "<api key>";
+
+const documents = [
```

Comment:
I don't think these documents are the best example to be used here. Could we get a list of documents in different languages? I think the ones in the language detection sample should suffice.